eras/babbage/test-suite/cardano-ledger-babbage-test.cabal

```
@@ -32,16 +32,16 @@ library
 
     build-depends:
         base >=4.14 && <4.19,
-        cardano-ledger-allegra >=1.1,
-        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
-        cardano-ledger-alonzo-test >=1.1,
-        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} >=1.5 && <1.7,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
-        cardano-ledger-shelley-ma-test >=1.1,
-        cardano-ledger-mary >=1.4,
-        cardano-ledger-shelley-test >=1.1,
-        cardano-ledger-shelley >=1.6 && <1.10,
+        cardano-ledger-allegra ^>=1.3,
+        cardano-ledger-binary:{cardano-ledger-binary, testlib} ^>=1.3,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} ^>=1.6,
+        cardano-ledger-alonzo-test ^>=1.1.2.9,
+        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} ^>=1.6,
+        cardano-ledger-core:{cardano-ledger-core, testlib} ^>=1.10,
+        cardano-ledger-shelley-ma-test ^>=1.2.1.6,
+        cardano-ledger-mary ^>=1.5,
+        cardano-ledger-shelley-test ^>=1.3.0.1,
+        cardano-ledger-shelley ^>=1.9,
```

There was not a single change to source code in this package. This is an indicator that we only need to relax bounds, not restrict them more, because this package will continue working with all of the older package versions.
```suggestion
        cardano-ledger-allegra >=1.1,
        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
        cardano-ledger-alonzo-test >=1.1,
        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} >=1.5 && <1.7,
        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
        cardano-ledger-shelley-ma-test >=1.1,
        cardano-ledger-mary >=1.4,
        cardano-ledger-shelley-test >=1.1,
        cardano-ledger-shelley >=1.6 && <1.10,
```